modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -336,6 +336,9 @@ def process_circuits(
             basebody["group"] = group
 
         handle_list = []
+        # set some default max-batch-cost
```

Comment:
The variable definitions just after this don't seem to relate to max-batch-cost - is this a TODO comment?